src/compiler/commandLineParser.ts

```diff
@@ -2541,7 +2542,16 @@ namespace ts {
         for (const id in jsonOptions) {
             const opt = optionNameMap.get(id);
             if (opt) {
-                defaultOptions[opt.name] = convertJsonOption(opt, jsonOptions[id], basePath, errors);
+

```

Comment:
This change is incorrect... You want do want to use convertJsonOption as was earlier here and combine the basePath with the nonrooted Paths as this is an API entry point and you cannot use sys.getCurrentDirectory() here.. 

The suggestedChange I recommended was to call `convertOptionsFromJson` with `basePath` as `sys.getCurrentDirectory()`